src/Language/PureScript/Docs/RenderedCode/Types.hs

```diff
@@ -159,7 +161,10 @@ syntax :: String -> RenderedCode
 syntax x = RC [Syntax x]
 
 ident :: String -> RenderedCode
-ident x = RC [Ident x]
+ident x = RC [Ident x ThisModule]
+
+ident' :: String -> ContainingModule -> RenderedCode
+ident' x m = RC [Ident x m]
```

Comment:
Yeah exactly. I just thought I'd mention this one too so it was on your radar, as it's probably a wider change than adjusting the sum type for aliases was.